cnxarchive/__init__.py

```diff
@@ -77,6 +77,8 @@ def add_route(name, path, *args, **kwargs):
     add_route('legacy-redirect-w-version', '/content/{objid}/{objver}{ignore:(/)?}{filename:(.+)?}')  # noqa cnxarchive.views:redirect_legacy_content
     add_route('recent', '/feeds/recent.rss')  # cnxarchive.views:recent
     add_route('oai', '/feeds/oai')  # cnxarchive.views:oai
+    add_route('xpath-book', '/xpath/book/{ident_hash}')
+    add_route('xpath-page', '/xpath/page/{ident_hash}')
```

Comment:
Just wondering if these could be combined into 1 route: `/xpath?id=${book_or_page_uuid}`. This might require looking up the `uuid` type (Book/Page) before executing the proper SQL query